qiskit/openquantumcompiler.py

```diff
@@ -0,0 +1,165 @@
+import qiskit.qasm as qasm
+import qiskit.unroll as unroll
+import qiskit.mapper as mapper
+
+import pdb
+import os
+import sys
+
+class ForkablePdb(pdb.Pdb):
```

Comment:
Is this class meant to be a general util for debugging, or a helper used on your local environment? I couldn't find any usages of it in the current codebase, but looks handy - if it is the first case, maybe it can be moved to a `helpers` module as it seems it can be useful in a more general sense?

Reply:
Not sure, @ewinston, What was the purpose of this class?